scripts/compatibility_matrix.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env python3
+
+# License: CC0 https://creativecommons.org/share-your-work/public-domain/cc0/
+
+# A messy script that figures out ZFS features. It's very messy, sorry. I am
```

Comment:
I would remove this bit and probably the bit about man pages. (If you keep that bit, let's remove the "lazy" bit.)